src/main/resources/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate/config.jelly

```diff
@@ -327,5 +327,8 @@
         </table>
       </f:repeatable>
     </f:entry>
+    <f:entry title="${%SessionTags}">
```

Comment:
Add a new field to the cloud config to toggle session taggyness vs no session taggyness

Reply:
good thinking